go/tools/builders/checker_main.go

```diff
@@ -90,44 +84,52 @@ func run(args []string) error {
 	return nil
 }
 
-// parseArchivesAndImportMap parses the -archivefile and -importmap arguments
-// created by the compile rule into the packageFiles and importMap maps used
-// by the vet and checker importers.
-func parseArchivesAndImportMap(archiveFiles, importMapIn []string) (packageFiles map[string]string, importMap map[string]string, err error) {
+// parseImportCfg parses the importcfg file generated by the compile program
+// and produces the packageFiles and importMap maps used by the vet and checker
+// importers.
+func parseImportCfg(importcfg string) (packageFiles map[string]string, importMap map[string]string, err error) {
```

Comment:
You may want to copy `readImportCfg` from `$GOROOT/src/cmd/compile/internal/gc/main.go`. This parser works fine for the files generated by our compile.go, but it doesn't handle comments or extra whitespace, so it's possible we may be break it in the future.

Reply:
Done.